content/departments/product-engineering/engineering/cloud/devops/managed/v1.1/mi1-1_restore_process.md

```diff
@@ -0,0 +1,84 @@
+# Restore a managed instance v1.1 when it is completely gone
+
+<span class="badge badge-note">SOC2/CI-110</span>
+
+> Are you experiencing a zonal failure? Follow the [failover process](./mi1-1_failover_process.md)
+
+In MI v1.1, we now have resources running outside the VM, i.e Cloud SQL, hence the restoration comes with multiple stages. Perform the restoration in the following order.
+
+## Restoring Cloud SQL
+
+Use cases:
+
+- Cloud SQL data is corrupted by a broken database migration
+- Cloud SQL data is deleted
+
+### Restore from automated backup
+
+> Below process is derived from [GCP documentation](https://cloud.google.com/sql/docs/postgres/backup-recovery/restoring#gcloud)
+
+The restortion process will be performed with `gcloud`. Learn more about [why not terraform?](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#restore_backup_context).
+
+Locate the SQL instance, note the name of the instance as `SQL_INSTANCE`
+
+```sh
+gcloud sql instances list --project $PROJECT_ID
+```
+
+List all backups, note the name of the latest (or the one right before database state is corrupted) **SUCCESSFUL** backup as `SQL_BACKUP_ID`
+
+```sh
+gcloud sql backups list --instance $SQL_INSTANCE --project $PROJECT_ID
+```
+
+Restore the backup to the current instance. **IMPORTANT** This will override all current data with the backup.
```

Comment:
Renders with a bit more visibility with the blue info box

```suggestion
Restore the backup to the current instance.
> NOTE: **IMPORTANT** This will override all current data with the backup.
```